partner_manual_rank/views/res_partner.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_res_partner_filter" model="ir.ui.view">
+        <field name="name">partner_manual_rank.view_partner_filter</field>
+        <field name="inherit_id" ref="base.view_res_partner_filter" />
+        <field name="model">res.partner</field>
+        <field type="xml" name="arch">
+            <xpath expr="//filter[@name='inactive']" position="before">
+                <filter
+                    string="Customers"
+                    name="customers"
+                    domain="[('is_customer', '=', True)]"
+                />
+                <filter
+                    string="Suppliers"
+                    name="suppliers"
+                    domain="[('is_supplier', '=', True)]"
+                />
+            </xpath>
+        </field>
+    </record>
+    <record id="view_res_partner_filter_account" model="ir.ui.view">
+        <field name="name">partner_manual_rank.view_partner_filter.account</field>
+        <field name="inherit_id" ref="account.res_partner_view_search" />
+        <field name="model">res.partner</field>
+        <field type="xml" name="arch">
+            <filter name="customer" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </filter>
+            <filter name="supplier" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </filter>
+        </field>
+    </record>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">partner_manual_rank.view_partner_form</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="model">res.partner</field>
+        <field type="xml" name="arch">
+            <xpath expr="//field[@name='child_ids']/kanban/templates" position="before">
+                <field name="is_customer" />
+                <field name="is_supplier" />
+            </xpath>
+            <xpath expr="//field[@name='child_ids']/form/sheet" position="inside">
+                <field name="is_customer" invisible="1" />
+                <field name="is_supplier" invisible="1" />
```

Comment:
Are the `invisible="1"` attributes still required here  ? Taking into account https://github.com/odoo/odoo/pull/137031